test/support/test_helpers.ex

```diff
@@ -36,6 +37,27 @@ defmodule RemoteRetro.TestHelpers do
     Map.put(context, :user, user)
   end
 
+  def persist_other_user_for_retro(context) do
+    context = Map.merge(%{other_user: @other_user}, context)
+    %{other_user: other_user} = context
+    user_params = User.build_user_from_oauth(other_user)
+    other_user =
+      User.changeset(%User{}, user_params)
+      |> Repo.insert!
+
+    Map.put(context, :other_user, other_user)
+  end
+
+  def assign_idea(%{other_user: other_user, retro: retro} = context) do
+    idea = %RemoteRetro.Idea{assignee_id: other_user.id, body: "blurgh", category: "action-item", retro_id: retro.id, user_id: other_user.id} |> RemoteRetro.Repo.insert!
```

Comment:
Hey Nichol, tiny but helpful elixir tidbit is that you can leverage the `alias` macro (see comment) above so that you don't have to include the `RemoteRetro` module prefix repeatedly. To learn more, [peep this!](https://elixir-lang.org/getting-started/alias-require-and-import.html#alias)

Reply:
👍